### PR TITLE
developing system templates locally and new Helm function 

### PR DESCRIPTION
**Pull request by nitind123**
PIPE-10545 - Allow developing system templates locally and PIPE-10390 - Add Helm support yaml

### CODE REVIEW

**Review by Varsha-Arun — CHANGES_REQUESTED** [review]

**Comment by Varsha-Arun** [comment]
send controlplane pr and green build so we can get some context. please next time divide the changes and its hard to review.

File: helmFile10390-1/.jfrog-pipelines/.jfrog-pipelines/P_SYC_Pipelines_10390_file.yml [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+    steps:
+      - name: S_files
+        type: Bash
+        configuration:
+          environmentVariables:
+            env1: gitBranch
+            env2: sourceRepository
+        execution:
+          onExecute:
+            - echo {{ get (.Files.Get "config.yml" | fromYaml) "message" }}
```
[the remark this answers is not in the document]
Comment (nitind123) [reply helmFile10390-1/.jfrog-pipelines/.jfrog-pipelines/P_SYC_Pipelines_10390_file.yml:13]: Most of them required SCM based pipeline only ... but for other I will take a note for next time

File: tests/yaml/values/syncTemplate/values.yml [hunk 2]
```diff
@@ -7,12 +7,12 @@ Pipeline:
...
   name: PS_Sync_template_GitRepo_10097
   gitProvider: s_gitHub
-  path: jfbetapipeorg/Org_test_Automation
+  path: {{.jfrog-pipelines.sourceRepository}}
   branches:
     exclude: 'master'
     include: {{gitBranch}}
 
-artifactoryIntegration: art
+artifactoryIntegration: s_artifactory
```
[the remark this answers is not in the document]
Comment (nitind123) [reply tests/yaml/values/syncTemplate/values.yml:15]: https://git.jfrog.info/projects/PIP/repos/controlplane/pull-requests/3449/diff#automation/tests/core/pipelines/SCM/pipelineSources.js

[the remark this answers is not in the document]
Comment (nitind123) [reply tests/yaml/values/syncTemplate/values.yml:15]: noted -- will break in multiple PRs

[the remark this answers is not in the document]
Comment (nitind123) [reply tests/yaml/values/syncTemplate/values.yml:15]: To get green build need changes available  in jfbetapipeorg/Org_test_Automation repo

**Comment by Varsha-Arun** [comment]
ok merging please fix if you see any failures

**Pull request merged**